src/sound/sound_server.cpp

```diff
@@ -229,10 +231,18 @@ static bool External_Volume(int volume, int oldVolume) {
 	return false;
 }
 #else
-#define External_Play(file) false
-#define External_IsPlaying() false
-#define External_Stop() false
-#define External_Volume(volume, oldVolume) false
+static bool External_Play(const std::string &file) {
+	return false;
+}
+static bool External_IsPlaying() {
+	return false;
+}
+static bool External_Stop() {
+	return false;
+}
+static bool External_Volume(int volume, int oldVolume) {
```

Comment:
```suggestion
static bool External_Volume(int /*volume*/, int /*oldVolume*/) {
```
Else, might produce warning for unused parameters.

Reply:
fixed